LoggerProj/Logger/FileLogger.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.IO;
+
+namespace Logger
+{
+    public class FileLogger : BaseLogger
+    {
+        private string filePath;
```

Comment:
Fields should be named with underscore PascalCase
```suggestion
        private string _FilePath;
```